### PR TITLE
Feature/virtualboxfixsb

### PR DESCRIPTION
**Pull request by inss-mark**
Revert to upsteam ubuntu packaged virtualbox to avoid issues with secureboot

### CODE REVIEW

File: roles/virtualbox/tasks/main.yml [hunk 1]
```diff
@@ -1,27 +1,5 @@
...
+- name: Install VirtualBox
   apt:
-    name: "{{ item }}"
-    state: absent
-  with_items:
-    - virtualbox
-
-- name: Configure the Oracle APT key
-  apt_key:
-    url: https://www.virtualbox.org/download/oracle_vbox_2016.asc
```
Comment (wasimj) [inline roles/virtualbox/tasks/main.yml]: What is the Oracle APT key used for?

Comment (inss-mark) [reply roles/virtualbox/tasks/main.yml]: Its the signing key for the virtualbox PPA package

Comment (wasimj) [reply roles/virtualbox/tasks/main.yml]: I should have said - what was it used for **and why do we not need it now**...?

Comment (wasimj) [reply roles/virtualbox/tasks/main.yml]: test

Comment (inss-mark) [reply roles/virtualbox/tasks/main.yml]: Full details are in the commit message body.

**Pull request merged**